sys/compat/freebsd64/freebsd64_misc.c

```diff
@@ -1414,7 +1414,7 @@ freebsd64___sysctlbyname(struct thread *td, struct
 	error = kern___sysctlbyname(td, __USER_CAP(uap->name, uap->namelen),
 	    uap->namelen, __USER_CAP(uap->old, oldlen),
 	    __USER_CAP_OBJ(uap->oldlenp), __USER_CAP(uap->new, uap->newlen),
-	    uap->newlen, &rv, 0, 0);
+	    uap->newlen, &rv, SCTL_MASK64, 1);
```

Comment:
I'm not sure what the correct value for the `inkernel` parameter is, I just copied this from freebsd32

Reply:
For `inkernel==1` we should be passing `&oldlen` instead of `__USER_CAP_OBJ(uap->oldlenp)` and updating the comment to remove the mention of the now non-existent race.